azure-pipelines-ci.yml

```diff
@@ -62,7 +62,7 @@ stages:
             displayName: "Run bandit"
           - script: |
               . venv/bin/activate
-              pre-commit run isort --all-files --show-diff-on-failure
+              pre-commit run isort --all-files --show-diff-on-failure --profile black
```

Comment:
This should not have been merged.

This is a change configuration in the CI. The right place to change this is in the configuration, which already has this profile.